pkg/collector/corechecks/snmp/internal/report/report_bandwidth_usage.go

```diff
@@ -135,7 +135,7 @@ func (ms *MetricSender) sendIfSpeedMetric(symbolName string, customSpeed uint64,
 	speedSource := "custom"
```

Comment:
this is when the `in/out_speed` is configured by the customer ?

Reply:
@NouhaManai96 yes, that's right :)